src/Task2/TimeOfYearByNumberOfMonth.java

```diff
@@ -0,0 +1,24 @@
+package Task2;
+
+import java.util.Scanner;
+
+public class TimeOfYearByNumberOfMonth {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Enter the month number: ");
+        int monthNumber = in.nextInt();
+        if ((monthNumber == 12) || (monthNumber == 1) || (monthNumber == 2)) {
```

Comment:
```suggestion
        if (monthNumber == 12 || monthNumber == 1 || monthNumber == 2) {
```
In this case (and below) we could omit this brackets